reco_utils/azureml/azureml_designer_modules/entries/score_sar_entry.py

```diff
@@ -0,0 +1,115 @@
+import argparse
```

Comment:
Please note score module in this implementation can be deployed as real-time inference. Currently real-time inference has many constraints on custom module. You can take this implementation as example if you're interested https://github.com/hirofumi-s-friends/recommenders/blob/weda/sar_ds_old/azureml-designer-modules/sar/score_sar_entry.py

Reply:
we decided not to implement this on this PR